monitoring/terraform/stack/grafana/main.tf

```diff
@@ -66,7 +66,7 @@ module "task" {
 }
 
 module "service" {
-  source = "github.com/wellcomecollection/terraform-aws-ecs-service.git//service?ref=v1.4.0"
```

Comment:
Why has the service moved internal to the module? I'm guessing the version used here is wildly out of date and upgrading is impractical?

Reply:
Pretty much, yeah. The actual motivation was tagging, explained here: https://github.com/wellcomecollection/platform-infrastructure/pull/223/files#diff-7d3d8ee04d5c20fcde3b70bb66fb6ecd3691fdd3bd00c6ecd82ef66d164e6b8cR61-R74

The Grafana service is very old and doesn't support tagging. We should upgrade it to a newer version, probably, but it works and I have no particular desire to fiddle with it.